src/frontend/package.json

```
@@ -32,6 +32,6 @@
     "node-sass": "4.11.0",
     "nodemon": "1.18.10",
     "prettier": "1.16.4",
-    "richie-education": "^1.8.3"
+    "richie-education": "^1.9.2"
```

```suggestion
    "richie-education": "1.9.2"
```

And you'll need to re-run `yarn install` to update the lock file to this change :)

---

Ok done !